src/main/java/apoc/warmup/Warmup.java

```diff
@@ -39,13 +39,23 @@ public Warmup() {
         int relsPerPage = pageSize / RelationshipRecordFormat.RECORD_SIZE;
         long nodesTotal = Util.nodeCount(db);
         long relsTotal = Util.relCount(db);
+        long maxNodeId = Util.runNumericQuery(db, "MATCH (n) RETURN max(id(n)) AS result", null);
```

Comment:
Oh, OK, I only looked at the merged version, not at the original PR. Sure, with the store providing that information, it's obviously faster.

I can create a version closer to the original, while still replacing the hard-coded values with constants.